walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingAsyncAwait()">Fetch a quote!</button>
```

Comment:
`getRandomQuoteUsingAsyncAwait` isn't defined anywhere I can see in the pull request.

Reply:
It looks like it is in this path location.
file:///home/muriloogps/software-product-sprint/walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/script.js